commands/makemessage.js

```diff
@@ -0,0 +1,31 @@
+const Discord = require("discord.js")
+
+module.exports = async (bot, message, args) => {
+    if (!args[0] || args[0] == "help"){
+        let help = new Discord.RichEmbed()
+        .setColor("#FFFF00")
+        .addField("Usage", "makemessage <channe_id> <message content>")
+        return message.channel.send(help)
+    }
+    if(!message.member.hasPermission("MANAGE_ROLES")){
+        message.reply("You don't have permission").then(async m=>{
+            await message.delete()
+            return m.delete(15000)
+        })
+    }
+    channel = message.guild.channels.find(f=>{
```

Comment:
Unless necessary (i.e. to avoid rebinding of this) arrow functions are discouraged for anything more than a quick one-line filter or transform. Avoid arrow functions for anything more than a one-liner.

```suggestion
    channel = message.guild.channels.find(function(f) {
```